2023-06-16T01:43:07.5215329Z ##[group]Run actions/setup-python@v4
2023-06-16T01:43:07.5215581Z with:
2023-06-16T01:43:07.5215782Z   python-version: 3.8
2023-06-16T01:43:07.5215987Z   architecture: x64
2023-06-16T01:43:07.5216181Z   cache: pip
2023-06-16T01:43:07.5216382Z   check-latest: false
2023-06-16T01:43:07.5216741Z   token: ***
2023-06-16T01:43:07.5216944Z   update-environment: true
2023-06-16T01:43:07.5217198Z   allow-prereleases: false
2023-06-16T01:43:07.5217409Z ##[endgroup]
2023-06-16T01:43:07.6859232Z ##[group]Installed versions
2023-06-16T01:43:07.6950548Z Successfully set up CPython (3.8.17)
2023-06-16T01:43:07.6950991Z ##[endgroup]
2023-06-16T01:43:08.2849845Z [command]/opt/hostedtoolcache/Python/3.8.17/x64/bin/pip cache dir
2023-06-16T01:43:08.6730704Z /home/runner/.cache/pip
2023-06-16T01:43:08.8018201Z pip cache is not found
